src/views/shared/hooks/use-list-workflows.ts

```diff
@@ -4,38 +4,49 @@ import { useMemo } from 'react';
 import { useInfiniteQuery } from '@tanstack/react-query';
 import queryString from 'query-string';
 
-import usePageQueryParams from '@/hooks/use-page-query-params/use-page-query-params';
 import {
+  type RouteParams as ListWorkflowsRouteParams,
   type ListWorkflowsResponse,
   type ListWorkflowsRequestQueryParams,
 } from '@/route-handlers/list-workflows/list-workflows.types';
 import request from '@/utils/request';
 import { type RequestError } from '@/utils/request/request-error';
-import domainPageQueryParamsConfig from '@/views/domain-page/config/domain-page-query-params.config';
 
-import DOMAIN_WORKFLOWS_PAGE_SIZE from '../config/domain-workflows-page-size.config';
-import { type UseListWorkflowsParams } from '../domain-workflows.types';
+import { type WorkflowsFiltersValues } from '../workflows-header/workflows-header.types';
```

Comment:
This type doesn't really make sense now that I reorganized the code, but where can we have it now?

Reply:
What is the exact issue with the type?

Reply:
Initially, I was calling useListWorkflows within the header itself, so I was passing the required query params values as a prop (that is what the type was for). But then I decided it made more sense to keep that logic outside the header.